### DISCUSSION TITLE
Conectar con Vercel

### DISCUSSION THREAD

**Discussion by Aiparraguirre2023** [community/community#61023] [New to GitHub]
Buenas, estoy empezando a usar github, quise asociar mi cuenta en vercel.com, pero cuando pongo conectar con github, me pone cuenta no encontrada. Ya verifique todos los permisos y está todo ok. aun asi...no hay forma de asociar las cuentas. Alguien podrá ayudarme?

**Comment by queenofcorgis** [comment]
Hi there, 

It looks like there may be some GitHub login issues per [Vercel's status page](https://www.vercel-status.com/incidents/590pcktngq7z). However, in the future, if status is showing green we recommend opening a support ticket up with Vercel to assist. 

Cheers!

**Reply by AntonioPasan07** [reply]
Yo tengo el mismo problema estoy investigando a ver si doy con ello.

**Comment by RI270382** [comment]
Hola!
Yo tengo el mismo problema. Podéis indicarme como solucionarlo? 
Gracias!
